src/targetValidationUtils.js

```diff
@@ -78,3 +84,25 @@ function isOverBoundaryEventTarget(clientX, clientY, paper, graph) {
       return component && validBoundaryEventTargets.includes(component.node.definition.$type);
     });
 }
+
+function isOverValidBoundaryErrorEventTarget(clientX, clientY, paper, graph) {
+  const target = graph
+    .findModelsFromPoint(getLocalMousePosition(clientX, clientY, paper))
+    .find(({ component }) => component && validBoundaryEventTargets.includes(component.node.definition.$type));
+
+  if (!target  || getAttachedErrorBoundaryEvents(target).length > 0) {
```

Comment:
Are there two spaces after `target` or is that just github?

Reply:
Nope, it's an extra space! 😨